contracts/main/stablecoin-core/LiquidationEngine.sol

```diff
@@ -107,11 +107,26 @@ contract LiquidationEngine is PausableUpgradeable, ReentrancyGuardUpgradeable, I
     address[] calldata _collateralRecipients,
     bytes[] calldata datas
   ) external override nonReentrant whenNotPaused onlyWhitelisted {
+
```

Comment:
2.3.17 Array lengths are not validated in 
LiquidationEngine
SEVERITY WARNING
STATUS NEW
SEVERITY WARNING
audit fix
from 3rd audit report iteration

Reply:
from Final Audit issues report